src/axom/primal/operators/squared_distance.hpp

```diff
@@ -82,7 +82,7 @@ AXOM_HOST_DEVICE inline double squared_distance(const Point<T, NDIMS>& P,
 
   if(!B.isValid())
   {
-    return std::numeric_limits<double>::max();
+    return std::numeric_limits<T>::max();
```

Comment:
Since the function has a return type of `double`, I think it will be cast to a `double` by the compiler.

Was this changed because of compiler warning/error or because the doxygen says the return type is `std::numeric_limits<T>::max()`?

Reply:
I don't think we're doing any repetitious interleaving and un-interleaving.  I've changed the 2D and 3D coordinate arrays to 1D interleaved data, so conduit operations no longer un-interleave them.  We are doing a lot of copies though, and any copy we eliminate will be good!
The T-to-double change was to be consistent with other `squared_distance` methods, to make it "look right" and to match the doxygen comment.